docs/source/libvirt.rst

```diff
@@ -43,7 +43,7 @@ definition, the following options are available.
 |                    |       |          | * name (req)  | with an incremented value (eth0) |
 |                    |       |          | * ip          |                                  |
 |                    |       |          | * mac         | .. note:: Network must exist     |
-|                    |       |          |               |                                  |
+|                    |       |          | *static_ip_reserved |                                  |
```

Comment:
Because 'static_ip_reserved' is not standard functionality, I suggest to explain what it does and not only how to enable it.